pystac_client/item_search.py

```diff
@@ -550,7 +550,7 @@ def _format(c: Any) -> Collections:
 
     @staticmethod
     def _format_ids(value: Optional[IDsLike]) -> Optional[IDs]:
-        if value is None:
+        if not value:
```

Comment:
I went whole hog and checked for truthiness here. As I'm writing this, I _think_ I need to be more careful w.r.t. the iterator case.